README.md

```diff
@@ -24,65 +40,101 @@ The structure of this lab — where its files and folders are located — looks
     └── indexTest.js
 ```
 
-All labs will more or less have the same structure. (And non-lab lessons, for that matter, will still have CONTRIBUTING.md, LICENSE.md, and README.md files.)
+All labs will more or less have the same structure. (And non-lab lessons, for
+that matter, will still have CONTRIBUTING.md, LICENSE.md, and README.md files.)
+
+### Code Along
 
-### Code-along
-For now, open up `index.js` in your text editor. If you're using the Learn IDE, click the blue "Open" button in the top right hand corner of the lesson. If you open up that `js-basics-variables-lab/` directory, you'll see a list of files (along with a `test/` directory). Click `index.js`, and it will open in the editor.
+Open up `index.js` in your text editor. If you're using the Learn IDE, click the
+blue "Open" button in the top right hand corner of the lesson. If you open up
+that `js-basics-variables-lab/` directory, you'll see a list of files (along
+with a `test/` directory). Click `index.js`, and it will open in the editor.
 
 In `index.js`, you should see, well, nothing. We'll fix that soon.
 
-Now open up `test/indexTest.js`. Hey, there's something! What's all of this stuff doing?
+Now open up `test/indexTest.js`. Hey, there's something! What's all of this
```

Comment:
Sometimes I think we write with too many vague colloquial speech patterns like "We did the thing!", "Look at all this stuff!". As we expand our reach, we might want to be careful about that. I think it might become confusing. I think we can be more detailed while still getting across a playful tone. 

eg: "Hey, there's a lot of code in here! What does all of this do?"